projects/geo/src/lib/print/print.module.ts

```diff
@@ -9,7 +9,8 @@ import {
   MatInputModule,
   MatFormFieldModule,
   MatRadioModule,
-  MatCheckboxModule
+  MatCheckboxModule,
```

Comment:
I think that the MatCheckboxModule and MatRadioModule modules are no longer useful